WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,12 +1,12 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table do %>
+  <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
     <thead>
       <tr>
-        <th class="name">Name</th>
-        <th class="wca-id">WCA ID</th>
-        <th class="country">Citizen of</th>
+        <th class="name" data-sortable="true" data-field="name">Name</th>
```

Comment:
Why do you only set data-field on this column? Is that so the `sort_name` you set above can work?